docs/items/overrides.md

```diff
@@ -12,21 +12,25 @@ To add a property to an `Item`, simply call `addPropertyOverride` on it. The `Re
 Using Overrides
 ---------------
 
-The format of an override can be seen on the [wiki][], and a good example can be found in `model/item/bow.json`. For reference, here is a hypothetical example of an item with a `power` property. Notice that when you define a predicate, it applies to *all values __greater than or equal to__ the given value*. If the values have no match, the default is the current model.
+The format of an override can be seen on the [wiki], and a good example can be found in `model/item/bow.json`. For reference, here is a hypothetical example of an item with a `power` property. Notice that when you define a predicate, it applies to *all values __greater than or equal to__ the given value*. If the values have no match, the default is the current model.
 
+```json
+{
+  "parent": "item/generated",
+  "texture": {
+    "__comment": "Default",
+    "layer0": "examplemod:items/examplePartial"
+  },
+  "overrides": [
     {
-      "parent": "item/generated",
-      "texture": {
-        "layer0": "examplemod:items/examplePartial" // Default
+      "__comment": "Power >= 0.75",
```

Comment:
I hate everything about that change. Ah well
On Aug 10, 2016 10:02 AM, "howtonotwin" <notifications@github.com> wrote:

> In docs/items/overrides.md
> <https://github.com/MinecraftForge/Documentation/pull/58#discussion_r74249576>
> :
>
> >      {
> > -      "parent": "item/generated",
> > -      "texture": {
> > -        "layer0": "examplemod:items/examplePartial" // Default
> > +      "__comment": "Power >= 0.75",
>
> I'll switch them back.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/MinecraftForge/Documentation/pull/58/files/bf7fac0591bc6fda0480c25a59d32b1736eb6844..bd5979c3a0dc0653c41631460e1c99214da7d463#r74249576>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADk-8OtPPgM0Ekiuma88Nx1jakDCKg-sks5qedn0gaJpZM4JgkWS>
> .
>